copyparty/docker-compose.yml

```
@@ -0,0 +1,18 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: copyparty_web_1
+      APP_PORT: 3923
+
+  web:
+    image: ghcr.io/9001/copyparty-ac:1.19.17@sha256:4fed30b24b217378314e10ebfbc67ad6c02be89734e8735c0b65f0dd8e524e0b
+    volumes:
+      - ${APP_DATA_DIR}/config/config.conf:/cfg/config.conf:ro
+      - ${APP_DATA_DIR}/data/cp_data:/cfg/hists
+      - ${APP_DATA_DIR}/data/shared:/shr
+      - ${APP_DATA_DIR}/media:/media
```

We can store all types of data whether configuration files or media within a single data directory. Please also change the directory path accordingly
```suggestion
      - ${APP_DATA_DIR}/data/media:/media
```